src/test/java/com/epam/tc/hw7/AbstractBaseTest.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.tc.hw7;
+
+import static com.epam.jdi.light.driver.WebDriverUtils.killAllSeleniumDrivers;
+import static com.epam.jdi.light.elements.init.UIFactory.$;
+
+import com.epam.tc.hw7.utils.LoginFromProp;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+public class AbstractBaseTest {
+
+    @BeforeTest()
+    public void setUp() {
+        LoginFromProp.login();
+        $("//a[contains(@href, 'metals-colors.html')]").click();
```

Comment:
You should no use straightforward approach in order to get web element and PO model at once.
In this HW PO is appreciated, use only this way, please.